File: app/src/main/java/com/haoyu/app/activity/AboutUsActivity.java
package com.haoyu.app.activity;

import android.view.View;

import com.haoyu.app.base.BaseActivity;
import com.haoyu.app.lego.student.R;
import com.haoyu.app.view.AppToolBar;

import butterknife.BindView;

/**
 * 关于我们
 *
 * @author xiaoma
 */
public class AboutUsActivity extends BaseActivity {

    @BindView(R.id.toolBar)
    AppToolBar toolBar;

    @Override
    public int setLayoutResID() {
        return R.layout.activity_about_us;
    }

    @Override
    public void setListener() {
        toolBar.setOnLeftClickListener(new AppToolBar.OnLeftClickListener() {
            @Override
            public void onLeftClick(View view) {
                finish();
            }
        });
    }
}

File: app/src/main/java/com/haoyu/app/activity/AboutUsActivity.kt
package com.haoyu.app.activity

import com.haoyu.app.base.BaseActivity
import com.haoyu.app.lego.student.R
import com.haoyu.app.view.AppToolBar

/**
 * 创建日期：2018/1/12.
 * 描述:关于我们
 * 作者:xiaoma
 */
class AboutUsActivity : BaseActivity() {
    override fun setLayoutResID(): Int {
        return R.layout.activity_about_us
    }

    override fun initView() {
        val toolBar = findViewById<AppToolBar>(R.id.toolBar)
        toolBar.setOnLeftClickListener {
            finish()
        }
    }

}